ci/conda/recipes/morpheus/meta.yaml

```diff
@@ -68,6 +68,7 @@ outputs:
       run:
         # Runtime only requirements. This + setup.py is the definitive runtime requirement list
         - {{ pin_compatible('cuda-cudart', min_pin='x.x', max_pin='x') }}
+        - beautifulsoup4=4.12
```

Comment:
I feel like this is only used by a few node types. Should we have an include guard instead of adding it to every install?

Reply:
It's used by the `RSSController`, so yes, there is only a single stage in Morpheus proper which requires it. 

However, by this logic the same would apply to `feedparser`